src/Skeleton/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FunctionComponent } from 'react'
+
+import clsx from 'clsx'
+import styles from './skeleton.module.scss'
+
+interface Props {
+  /** Sets the skeleton-pulse style */
+  type?: 'text' | 'image' | 'button' | 'button-secondary'
+  /** Sets the width of the skeleton. */
+  width?: number | string | []
```

Comment:
remove support for the array type until you add functionality for it with the count prop

Reply:
This has been removed